### ISSUE TITLE
DO NOT USE this app. SPYWARE WARNING

### ISSUE THREAD

**Issue by ple1n** [dmitriy-ilchenko/QrAndBarcodeScanner#158]
https://github.com/wewewe718/QrAndBarcodeScanner/issues/144

**Comment by GL513** [comment]
If you use F-Droid you're fine.
So, just use F-Droid.

**Comment by nvllz** [comment]
Yeah, on F-Droid the app comes with no Internet permission. It shouldn't be able to send any external request on its own.

**Comment by ple1n** [comment]
> If you use F-Droid you're fine. So, just use F-Droid.

I currently use Obtainium, which gets releases from github directly

**Comment by SegoCode** [comment]
> > If you use F-Droid you're fine. So, just use F-Droid.
> 
> I currently use Obtainium, which gets releases from github directly

Do not use github until some pr similar to this one merge https://github.com/wewewe718/QrAndBarcodeScanner/pull/151 meanwhile download the fdroid version

**Comment by abhigit23** [comment]
I use Barcode Scanner by Atharok, UI is great, everything just works fine(including flash) with no data collection at all :-)

**Comment by Noitarud** [comment]
This is also an Android issue, i have to give permission to use camera, storage, etc, but no permissions requested for internet. Go Netguard (~2020 version) or some kind of firewall…
